src/Dapr.AspNetCore/DaprMvcBuilderExtensions.cs

```diff
@@ -35,7 +38,18 @@ public static IMvcBuilder AddDapr(this IMvcBuilder builder)
                 return builder;
             }
 
-            builder.Services.AddDaprClient();
+            // create default clientbuilder config (if not specified)
+            if (configure == null)
+            {
+                configure = new Action<DaprClientBuilder>(
+                    builder => builder.UseJsonSerializationOptions(
```

Comment:
Better to move the default JsonSerializationOptions to DaprClient instead. That will ensure that the case insensitive options is always set, even when the user directly calls `AddDaprClient` instead of `AddDapr`. It also enables us to use case insensitive as the default options even when the user does pass a configure action to `AddDaprClient`.